pkg/model/trafficpatterns/ramp.go

```diff
@@ -65,7 +66,7 @@ func (r *ramp) Generate() {
 	}
 }
 
-func NewRamp(env simulator.Environment, source model.TrafficSource, buffer model.RequestsBufferedStock, config RampConfig) Pattern {
+func NewRamp(env simulator.Environment, source model.TrafficSource, buffer model.RequestsRoutingStock, config RampConfig) Pattern {
```

Comment:
Rename `buffer`.

Reply:
Renamed to routingStock in https://github.com/pivotal/skenario/pull/67/commits/dfd4d60851eea3b87cc9d7e4dc950b95bc92c495 commit.